src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3951,19 +3951,25 @@ def test_sql_server_trust_groups(self):
 
 class SqlManagedInstanceMgmtScenarioTest(ScenarioTest):
 
+    DEFAULT_MC = "SQL_Default"
+    MMI1 = "SQL_WestEurope_MI_1"
+
+    def _get_full_maintenance_id(self, name):
+        return "/subscriptions/{}/providers/Microsoft.Maintenance/publicMaintenanceConfigurations/{}".format(self.get_subscription_id(), name)
```

Comment:
Is it possible to create the resource on demand instead of hard code it?

Reply:
This is a static resource already defined and available for user to choose from a list. It's a public maintenance policy with defined schedule which when applied to a resource (in this case managed instance) makes sure that it get's updates according to policy schedule. These kind of policies are predefined and can't be created by a user. User can only get a list of those via MRP API and then apply one to a managed instance.